TwilioVerify/TwilioVerify/Sources/Domain/Manager/TwilioVerify.swift

```diff
@@ -166,7 +166,7 @@ public class TwilioVerifyBuilder {
     networkProvider = NetworkAdapter()
     jwtGenerator = JwtGenerator(withJwtSigner: JwtSigner())
     authentication = AuthenticationProvider(withJwtGenerator: jwtGenerator)
-    self.baseURL = Constants.baseURL
+    self._baseURL = baseURL
```

Comment:
Is this change needed?

Reply:
Yes, because baseURL is a global variable in `TwilioVerifyConfig.swift`